static/app/views/settings/project/projectToolbar.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import {ProjectFixture} from 'sentry-fixture/project';
+
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import ProjectToolbar from 'sentry/views/settings/project/projectToolbar';
+
+describe('ProjectToolbar', function () {
+  const {routerProps, organization, project, router} = initializeOrg();
+  const url = `/projects/${organization.slug}/${project.slug}/`;
+
+  beforeEach(function () {
+    MockApiClient.clearMockResponses();
+    MockApiClient.addMockResponse({
+      url,
+      method: 'GET',
+      body: ProjectFixture(),
+    });
```

Comment:
When/where does this get called? should we add an expectation for it?

Reply:
Adding a test for this!

Reply:
updated the tests. We don't need to mock this method for this component